DSTV.Net/Data/Dstv.cs

```diff
@@ -22,7 +22,8 @@ public string ToSvg()
         try
         {
             Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture("en-GB");
-            return @$"<svg width=""{Header?.Length}"" height=""{Header?.ProfileHeight}"" xmlns=""http://www.w3.org/2000/svg"">{string.Concat(Elements.OrderByDescending(d => d is Contour).Select(d => d.ToSvg()))}</svg>";
+            return
+                @$"<svg width=""{Header?.Length}"" height=""{Header?.ProfileHeight}"" xmlns=""http://www.w3.org/2000/svg"">{string.Concat(Elements.OrderByDescending(d => d is Contour).Select(d => d.ToSvg()))}</svg>";
```

Comment:
Zullen we deze nog even met een viewbox doen, zodat je zelf controle hebt over de scaling?

```suggestion
                @$"<svg viewbox=""0 0 {Header?.Length} {Header?.ProfileHeight}"" xmlns=""http://www.w3.org/2000/svg"">{string.Concat(Elements.OrderByDescending(d => d is Contour).Select(d => d.ToSvg()))}</svg>";
```